documentation/library_guide/api_for_sycl_kernels/tested_standard_cpp_api.rst

```diff
@@ -450,15 +450,17 @@ C++ Standard API                      libstdc++  libc++     MSVC
 ``std::truncf``                       Tested     Tested     Tested
 ===================================== ========== ========== ==========
 
-These tests were done for the following versions of the standard C++ library:
+The testing was conducted for the following versions of the C++ standard libraries, with the help of the listed compilers:
 
-============================================= =============================================
-libstdc++ (GNU)                               Provided with GCC* 8.4.0, GCC 9.3.0,
-                                              GCC 11.4.0, GCC 13.2.0
---------------------------------------------- ---------------------------------------------
-libc++ (LLVM)                                 Provided with Clang* 11.0, Clang 14.0,
-                                              Clang 17.0, Clang 18.1
---------------------------------------------- ---------------------------------------------
-Microsoft Visual C++* (MSVC) Standard Library Provided with Microsoft Visual Studio 2019
-                                              and Microsoft Visual Studio 2022.
-============================================= =============================================
+================================== ============================================= =============================================
+SYCL Compiler                      Standard C++ Library                          Standard C++ Library Version
```

Comment:
I suggest putting the compiler into the last column and name it as "SYCL Compiler Used".

https://github.com/uxlfoundation/oneDPL/pull/2080#discussion_r2075161559 describes why.

Reply:
Done